### PR TITLE
openstack-test: Instrument branch release-4.15

### PR DESCRIPTION
**Pull request by pierreprinetti**

### CODE REVIEW

**Comment by pierreprinetti** [comment]
/pj-rehearse pull-ci-openshift-openstack-test-release-4.15-images

**Comment by pierreprinetti** [comment]
/pj-rehearse pull-ci-openshift-openstack-test-release-4.15-images

File: ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: rhel-9-release-golang-1.20-openshift-4.14
+    tag: rhel-9-release-golang-1.20-openshift-4.15
```
Comment (mandre) [inline ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml:6]: This should be 4.16, shouldn't it?

File: ci-operator/config/openshift/openstack-test/openshift-openstack-test-release-4.16.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: rhel-9-release-golang-1.20-openshift-4.14
+    tag: rhel-9-release-golang-1.20-openshift-4.15
```
Comment (mandre) [inline ci-operator/config/openshift/openstack-test/openshift-openstack-test-release-4.16.yaml:6]: ditto

File: core-services/prow/02_config/openshift/openstack-test/_prowconfig.yaml [hunk 3]
```diff
@@ -8,6 +8,7 @@ tide:
     - release-4.12
     - release-4.13
     - release-4.14
+    - release-4.15
```
Comment (mandre) [inline core-services/prow/02_config/openshift/openstack-test/_prowconfig.yaml:11]: There too, we should add the release-4.16 branch

Comment (pierreprinetti) [reply core-services/prow/02_config/openshift/openstack-test/_prowconfig.yaml:11]: I don't think so. We will only merge to release-4.16 once it's no longer synced to the main branch

File: ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml (hunk 1 above)
Comment (pierreprinetti) [reply ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml]: Installer uses 4.15... but let's see if 4.16 is available already.

**Comment by pierreprinetti** [comment]
/pj-rehearse pull-ci-openshift-openstack-test-release-4.16-build pull-ci-openshift-openstack-test-release-4.15-build

File: core-services/prow/02_config/openshift/openstack-test/_prowconfig.yaml (hunk 3 above)
Comment (mandre) [reply core-services/prow/02_config/openshift/openstack-test/_prowconfig.yaml:11]: You're right.

**Review by mandre — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**